src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+export type ErrorMessageProps = {
+  text?: string;
+  id?: string;
+  className?: string;
+  visuallyHiddenText?: string;
+  children?: ReactNode;
+};
+
+export const ErrorMessage: FunctionComponent<ErrorMessageProps> = ({
+  text,
+  id,
+  className,
+  visuallyHiddenText,
+  children
+}) => {
+  const messageClassName = classnames(
+    "govuk-error-message",
+    "lbh-error-message",
+    className
+  );
+  return (
+    <span id={id} className={messageClassName}>
+      {visuallyHiddenText && (
+        <span className="govuk-visually-hidden">{visuallyHiddenText}:</span>
+      )}
+      {children ? children : text}
+    </span>
+  );
+};
+
+ErrorMessage.propTypes = {
+  text: PropTypes.string,
+  id: PropTypes.string,
+  className: PropTypes.string,
+  visuallyHiddenText: PropTypes.string,
+  children: PropTypes.element
```

Comment:
```suggestion
  children: PropTypes.node
```
I wonder if this should actually replace `text` entirely. That feels more like the React Way and would simplify the component.

Reply:
Personally I like the separate `text` and `children` parameters as it matches what's in the govuk template. Could be persuaded otherwise though.

Reply:
So the thing for me is that there's no use case where you'd need both. It's more idiomatic React to use `children` even if it's only a text node that you pass in. It's more familiar to write `<ErrorMessage>my error message</ErrorMessage>` than `<ErrorMessage text="my error message" />`. That's my take.